### ISSUE TITLE
Latest Report column labels - allow personalization of units for Temp

### ISSUE THREAD

**Issue by davroth** [patrickomni/omnimobileserver#73]
Capturing this for completeness - is there any way to control units for temperature?

Much of the world uses centigrade while US uses Fahrenheit; mental computation/switching can be tricky. We should either allow setting by account or show both

**Comment by CaseyDengler** [comment]
I don't think that it's currently a feature, but the conversions are quite simple.

°F = °C  x  9/5 + 32
0 °C = 273.15 K

**Comment by patrickomni** [comment]
Added work item; moving to future.
